src/main/java/dev/slimevr/gui/TrackersList.java

```diff
@@ -235,7 +235,9 @@ public void actionPerformed(ActionEvent e) {
 			add(status = new JLabel(t.getStatus().toString().toLowerCase()), c(1, row, 2, GridBagConstraints.FIRST_LINE_START));
 			if(realTracker instanceof TrackerWithBattery) {
 				add(new JLabel("Battery:"), c(2, row, 2, GridBagConstraints.FIRST_LINE_START));
-				add(bat = new JLabel("0"), c(3, row, 2, GridBagConstraints.FIRST_LINE_START));
+				add(batlvl = new JLabel("0%"), c(3, row, 2, GridBagConstraints.FIRST_LINE_START));
+				add(new JLabel("Voltage:"), c(2, row, 3, GridBagConstraints.FIRST_LINE_START));
+				add(bat = new JLabel("0.0V"), c(3, row, 3, GridBagConstraints.FIRST_LINE_START));
```

Comment:
You have both labels and values in the same cell. Third argument of c() is padding.